dev_docs/tutorials/saved_objects.mdx

```diff
@@ -14,7 +14,7 @@ The folder should contain a file per type, named after the snake_case name of th
 **src/plugins/my_plugin/server/saved_objects/dashboard_visualization.ts**
 
 ```ts
-import { SavedObjectsType } from 'src/core/server';
+import { SavedObjectsType } from 'src/core/packages/core/server';
```

Comment:
All the imports here should actually be `@kbn/core/server` (or the individual saved-objects package)